### DISCUSSION TITLE
HttpError: 403 Forbidden

### DISCUSSION THREAD

**Discussion by Nathan-MV** [cawa-93/vite-electron-builder#945] [Q&A]
Does anyone know how to resolve?
![image](https://github.com/cawa-93/vite-electron-builder/assets/67521199/61a2d734-ee69-4c47-b1bd-6e74ee80f22c)
Error #1 --------------------------------------------------------------------------------
HttpError: 403 Forbidden
"method: post url: [https://api.github.com/repos/Nathan-MV/chaser/releases\n\n](https://api.github.com/repos/Nathan-MV/chaser/releases/n/n)          Data:\n          {\"message\":\"Resource not accessible by integration\",\"documentation_url\":\"https://docs.github.com/rest/releases/releases#create-a-release\"}\n          "

**Comment by cawa-93** [comment]
This may be related to the repository settings. For some time, GitHub by default does not allow for actions to make changes to the repository
